_widget/src/components/app/component.vue

```diff
@@ -44,6 +44,10 @@ export default {
       type: String,
       default: ''
     },
+    gettingStartedUrl: {
+      type: String,
+      default: '/articles/getting-started/'
```

Comment:
We're lucky there is no overlap of URLs between the sites. But this could be a potential issue in the future.

Reply:
Maybe we can modify `findArticle` to accept a source? That way it only returns an article with a matching `sourceUrl`.

```js
this.findArticle(this.gettingStartedUrl, 'https://support.dnsimple.com')
```

Reply:
Changed `findArticle` to just use the absolute URL: https://github.com/dnsimple/dnsimple-support/pull/1375/commits/0bf761b1c6167595794b4e18c693ed8ce1672a04